### DISCUSSION TITLE
Error in pointinpolygon syntax operation：【Max query size exceeded】

### DISCUSSION THREAD

**Discussion by oncream** [ClickHouse/ClickHouse#28010] [Q&A] [answered]
My query statement reported an error，position it's so big.
error:
 Syntax error: failed at position 262140 ('31.071081'): 31.071081),(106.859669,31.071675),(106.861177,31.072891),(106.863658,31.075149),(106.865878,31.077923),(106.869148,31.081165),(106.871906,31.083311),(106.874223. Max query size exceeded: '31.071081'"}
How can I solve it，
very urgent.thank you.

**Comment by alexey-milovidov** [comment, marked as answer]
Increase `max_query_size` setting.
